jina/optimizers/__init__.py

```diff
@@ -105,22 +107,24 @@ class FlowOptimizer(JAMLCompatible):
     """
 
     def __init__(
-        self,
-        flow_runner: 'FlowRunner',
-        parameter_yaml: str,
-        evaluation_callback: 'OptimizerCallback',
-        n_trials: int,
-        workspace_base_dir: str = '',
-        sampler: str = 'TPESampler',
-        direction: str = 'maximize',
-        seed: int = 42,
+            self,
+            flow_runner: 'FlowRunner',
+            parameter_yaml: str,
+            evaluation_callback: 'OptimizerCallback',
+            n_trials: int,
+            workspace_base_dir: str = '',
```

Comment:
Please don't do edit wars in indentation. I guess that happened due to auto code formatting, but we want to go towards black in the future anyhow, which will do the indentation that was here before.

Reply:
I saw, that quite some people started editing indentation at hanging parameters in function definitions to go 8 spaces deep instead of 4. So you can keep it, even thought I find it ugly.

Reply:
Yes! We have to do something against edit wars! I use the pep8 formatter. We could also all agree on using pep8. So we won't have different formatting.